### DISCUSSION TITLE
In Moonlight: Connection terminated: No video from host

### DISCUSSION THREAD

**Discussion by Tobanja** [VirtualDrivers/Virtual-Display-Driver#198] [Q&A]
I'm sorry if this has been asked, but I've been trying for hours to make streaming the virtual monitor to my TV (android based). It appears as if Moonlight can't see a signal to the virtual monitor. Everything has been set up and double checked, but I have the feeling I'm missing something basic here:

Installed the virtual driver and it shows up fine in windows. I set it to 1080p 60fps just to use conservative settings when testing.

In sunshine, I set the virtual monitor as output name, \\.\DISPLAY26. It should be noted, it changed name after a reboot. I hope it doesn't do that after every reboot.

I tested setting it back to my primary monitor, and then streaming works fine (as it always did).
I'm not even sure how I can test if the virtual device has a signal since I can't see it. I've however tried both to extend and to duplicate the display, same error.

What am I missing here? Using the latest sunshine and moonlight, Nvidia 4070 GPU. I've read somewhere that I might need to disable the integrated graphics for it to work (CPU AMD Ryzen 9 7900) . Might be worth a try?

Output from dxgi-info:

====== ADAPTER =====
Device Name      : NVIDIA GeForce RTX 4070
Device Vendor ID : 0x000010DE
Device Device ID : 0x00002786
Device Video Mem : 12012 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 32308 MiB

    ====== OUTPUT ======
    Output Name       : \\.\DISPLAY1
    AttachedToDesktop : yes
    Resolution        : 5120x1440

    Output Name       : \\.\DISPLAY26
    AttachedToDesktop : yes
    Resolution        : 1920x1080

====== ADAPTER =====
Device Name      : AMD Radeon(TM) Graphics
Device Vendor ID : 0x00001002
Device Device ID : 0x0000164E
Device Video Mem : 485 MiB
Device Sys Mem   : 0 MiB
Share Sys Mem    : 32308 MiB

    ====== OUTPUT ======
...

**Comment by bud3699** [comment]
It was a bit iffy with the driver for example if you set the driver as the main display apparently it works.

Also apparently this is fixed in the latest sunshine beta
